Tests/TuistGeneratorTests/Generator/BuildPhaseGeneratorTests.swift

```diff
@@ -86,6 +86,39 @@ final class BuildPhaseGeneratorTests: XCTestCase {
         }
     }
 
+    func test_generateSourcesBuildPhase_whenLocalizedFile() throws {
+        // Given
+        let target = PBXNativeTarget(name: "Test")
+        let pbxproj = PBXProj()
+        pbxproj.add(object: target)
+
+        let files: [AbsolutePath] = [
+           "/path/sources/Base.lproj/OTTSiriExtension.intentdefinition",
+           "/path/resources/en.lproj/OTTSiriExtension.intentdefinition",
+       ]
+
+       let fileElements = createLocalizedResourceFileElements(for: [
+           "/path/resources/OTTSiriExtension.intentdefinition",
+       ])
+
+        // When
+        try subject.generateSourcesBuildPhase(files: sources,
+                                              pbxTarget: target,
+                                              fileElements: fileElements,
+                                              pbxproj: pbxproj)
+
+        // Then
+        let buildPhase = try target.sourcesBuildPhase()
+        let buildFiles = buildPhase?.files ?? []
+        let buildFilesNames = buildFiles.map {
+            $0.file?.name
+        }
+
+        XCTAssertEqual(buildFilesNames, [
```

Comment:
I believe this may need to be:

```swift
XCTAssertEqual(buildFiles.map { $0.file }, [
        fileElements.elements["/path/sources/OTTSiriExtension.intentdefinition"],
])
```

Reply:
Thank you, fixed.